README.md

```diff
@@ -14,66 +14,141 @@ Takeaway Challenge
 
  ```
 
-Instructions
--------
 
-* Challenge time: rest of the day and weekend, until Monday 9am
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+(updated: 14 Oct 2018 - COVERAGE:  84.17% -- 101/120 lines in 6 files)
 
-Task
------
+Installation
+-------
 
-* Fork this repo
-* Run the command 'bundle' in the project directory to ensure you have all the gems
-* Write a Takeaway program with the following user stories:
+1. Fork this repo, and clone to your local machine
+1. When the installation completes, run `bundle`
 
-```
-As a customer
-So that I can check if I want to order something
-I would like to see a list of dishes with prices
 
-As a customer
-So that I can order the meal I want
-I would like to be able to select some number of several available dishes
+Usage
+---------
 
-As a customer
-So that I can verify that my order is correct
-I would like to check that the total I have been given matches the sum of the various dishes in my order
+Here's how you might explore the application using something like pry:
 
-As a customer
-So that I am reassured that my order will be delivered on time
-I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
 ```
-
```

Comment:
Good README with instructions but could add a bit about your approach to solving the challenge